index.rst

```diff
@@ -100,73 +110,268 @@ Data Sets Types and Goals
+  2. use the DECam Bulge survey for crowded field tests.  IN PROGRESS.
+  3. Select a subset of HSC SSP PDR1 vs PDR2.  TICKET OPEN.
+  4. use a DES Deep SN field for large-scale processing.
+
+Desiderata for AP testing:
+  - tens of epochs per filter per tract in order to construct templates for image differencing and to characterize variability
+  - the ability to exercise as many aspects of LSST pipelines and data products as possible
+  - public availability (so that we can feely recruit various LSST stakeholders)
+  - potential for enabling journal publications (both technical and scientific) so that various stakeholders beyond LSST DM may have direct interest in contributing tools and analysis.
+  - We should have datasets from at least two different cameras, so that we can isolate effects of LSST pipeline performance from camera-specific details (e.g., ISR, PSF variations) that impact the false-positive rate
+  - at least one dataset should be from HSC, to take advantage of Princeton's work on DRP processing
+  - at least one dataset should be from a camera without an ADC to test DCR.  * This is currently unmet *.
+  - probably only two cameras should be used for regular detailed processing, to avoid spending undue DM time characterizing non-LSST cameras.  HSC and DECam are the clear choices for this, but do not satisfy the no ADC optios.
+  - datasets should include regions of both high and low stellar densities, to understand the impact of crowding on image differencing
+  - ideally, data will be taken over multiple seasons to enable clear separation of templates from the science images
+  - datasets sampling a range of timescales (hours, days, ... years) provide the most complete look at the real transient and variable population
+  - datasets with multiple filters will aid in understanding our DCR performance
+  - substantial dithering or field overlaps will allow us to test our ability to piece together templates from multiple images (some transient surveys, such as HiTS, PTF, and ZTF, use a strict field grid)
+  - there is a balance to be struck between using datasets that have been extensively mined scientifically by the survey times as opposed to datasets that have not been exploited completely.  If published catalogs of variables, transients, and/or asteroids exist, they will aid in false-positive discrimination and speed QA work.  On the other hand well-mined datasets may be less motivating to work on, particularly for those outside LSST DM.
+  - LSST-like cadences to test Solar System Orbit algorithms
+
+CI
+==
+1. DECam HiTS
+    - A subset of data intended for CI AP testing (with Blind15A_40 and Blind15A_42) is in
+      https://github.com/lsst/ap_verify_ci_hits2015
+    This subset is only 3 visits and 2 CCDs per visit.
+    Presently (2018-08-15) the data are on a branch, not yet merged to master.
+
+SMALL
+=====
+1. DECam HiTS
+    - See https://dmtn-039.lsst.io/
```

Comment:
I would remove the link to this technote, which is wildly out of date and tends to confuse users who really want the `ap_pipe` documentation on `pipelines.lsst.io`.